quartz/components/scripts/popover.inline.ts

```diff
@@ -37,29 +37,49 @@ async function mouseEnterHandler(
   targetUrl.hash = ""
   targetUrl.search = ""
 
-  const contents = await fetch(`${targetUrl}`)
-    .then((res) => res.text())
-    .catch((err) => {
-      console.error(err)
-    })
+  const response = await fetch(`${targetUrl}`).catch((err) => {
+    console.error(err)
+  })
 
   // bailout if another popover exists
   if (hasAlreadyBeenFetched()) {
     return
   }
 
-  if (!contents) return
-  const html = p.parseFromString(contents, "text/html")
-  normalizeRelativeURLs(html, targetUrl)
-  const elts = [...html.getElementsByClassName("popover-hint")]
-  if (elts.length === 0) return
+  if (!response) return
+  const contentType = response.headers.get("Content-Type")
 
   const popoverElement = document.createElement("div")
   popoverElement.classList.add("popover")
   const popoverInner = document.createElement("div")
   popoverInner.classList.add("popover-inner")
   popoverElement.appendChild(popoverInner)
-  elts.forEach((elt) => popoverInner.appendChild(elt))
+
+  if (contentType && contentType.includes("image")) {
```

Comment:
can we gated this into an options?

Reply:
Do you mean like with cfg.enablePopovers? How would you want it implemented just added to the if on 58?

Reply:
Is show image popover a default obsidian behaviour?

If it is then I think we don't need an option for it.

Reply:
Yeah just tested without any of my plugins, popovers for image wikilinks is native

Reply:
ok, then no need for options then.

Quick ask, can you use match-case instead of if logic like this?

Reply:
Adjusted in 102af9e